qcore/constants.py

```diff
@@ -164,7 +164,33 @@ class ProcessType(ExtendedStrEnum):
     # ProcessID, ProcessName, "Does this task use Hyperthreading?", "Does this use an Acc directory?", command, dependancies (tuple),
     """
 
-    VM_PARAMS = 16, "VM_PARAMS", None, False, None, ([],)
+    # GCMT_2_REL = 24, "GCMT_2_REL", None, False, None, ([],)
+    # NHM_2_REL = 23, "NHM_2_REL", None, False, None, ([],)
```

Comment:
Do the commented out lines need to be here?

Reply:
They're there as pre-fill for automated realisation generation, to reduce the work later on.
Don't have to be there, but they show the way forward and I don't think it hurts to have them there